content/php/concepts/math-functions/terms/round/round.md

```diff
@@ -0,0 +1,60 @@
+# Title:     _round function in php_
+
+## Description
+    The round() function rounds a floating-point number.
+    
+## Tips:
+    | To round a number UP to the nearest integer, look at the ceil() function.
+    | To round a number DOWN to the nearest integer, look at the floor() function.
+    | Return type of round function is float
```

Comment:
[Proofreading] Since `float` describes a specific coding concept, consider putting it backticks. 
```suggestion
    | Return type of round function is `float`
```